joomla/joomla-cms
# [4.0] Password toggler markup and styling

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/24312.

### Summary of Changes

Corrects password toggler markup and styling to have a visible focus outline.
Converts to FA icons, adds `fa-fw` class to keep same width when toggling.

### Testing Instructions

Run `node build.js --compile-js`.
Go to a form with password field.
Inspect styling.
Click on toggle button a few times.
Check that the icon and the screenreader text changes correctly.

### Documentation Changes Required

No.
